src/main/resources/db/migration/V5_1__add_column_is_sendt_to_arena.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE DIALOGMELDINGOPPLYSNINGER
+    ADD COLUMN is_sent_to_arena BOOLEAN DEFAULT FALSE;
+
+UPDATE dialogmeldingopplysninger
+SET is_sent_to_arena = TRUE
+WHERE arena IS NOT NULL;
```

Comment:
Eller er motsatt bedre, `DEFAULT TRUE` og oppdatere `FALSE WHERE arena IS NULL`? Føler at `false` er en bedre defaultverdi, men hvis vi gjør det motsatt så blir updaten på L4 gjort på et mindre antall rader.

Reply:
Ja, synes det leser bedre sånn det er nå, så stemmer for det hvis vi tror at kjøretiden ikke blir noe problem :crossed_fingers: